<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>TUview</class>
 <widget class="QWidget" name="TUview" >
  <property name="enabled" >
   <bool>true</bool>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>280</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>13</hsizetype>
    <vsizetype>13</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize" >
   <size>
    <width>16777187</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="focusPolicy" >
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle" >
   <string>Detail</string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <property name="autoFillBackground" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item rowspan="3" row="0" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>16</width>
       <height>588</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="lblComment" >
     <property name="palette" >
      <palette>
       <active>
        <color>
         <red>255</red>
         <green>24</green>
         <blue>35</blue>
        </color>
        <color>
         <red>221</red>
         <green>223</green>
         <blue>228</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>85</red>
         <green>85</green>
         <blue>85</blue>
        </color>
        <color>
         <red>199</red>
         <green>199</green>
         <blue>199</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>239</red>
         <green>239</green>
         <blue>239</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>103</red>
         <green>141</green>
         <blue>178</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>238</blue>
        </color>
        <color>
         <red>82</red>
         <green>24</green>
         <blue>139</blue>
        </color>
        <color>
         <red>232</red>
         <green>232</green>
         <blue>232</blue>
        </color>
       </active>
       <inactive>
        <color>
         <red>255</red>
         <green>24</green>
         <blue>35</blue>
        </color>
        <color>
         <red>221</red>
         <green>223</green>
         <blue>228</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>85</red>
         <green>85</green>
         <blue>85</blue>
        </color>
        <color>
         <red>199</red>
         <green>199</green>
         <blue>199</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>239</red>
         <green>239</green>
         <blue>239</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>103</red>
         <green>141</green>
         <blue>178</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>238</blue>
        </color>
        <color>
         <red>82</red>
         <green>24</green>
         <blue>139</blue>
        </color>
        <color>
         <red>232</red>
         <green>232</green>
         <blue>232</blue>
        </color>
       </inactive>
       <disabled>
        <color>
         <red>128</red>
         <green>128</green>
         <blue>128</blue>
        </color>
        <color>
         <red>221</red>
         <green>223</green>
         <blue>228</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>85</red>
         <green>85</green>
         <blue>85</blue>
        </color>
        <color>
         <red>199</red>
         <green>199</green>
         <blue>199</blue>
        </color>
        <color>
         <red>199</red>
         <green>199</green>
         <blue>199</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>128</red>
         <green>128</green>
         <blue>128</blue>
        </color>
        <color>
         <red>239</red>
         <green>239</green>
         <blue>239</blue>
        </color>
        <color>
         <red>239</red>
         <green>239</green>
         <blue>239</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>86</red>
         <green>117</green>
         <blue>148</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>238</blue>
        </color>
        <color>
         <red>82</red>
         <green>24</green>
         <blue>139</blue>
        </color>
        <color>
         <red>232</red>
         <green>232</green>
         <blue>232</blue>
        </color>
       </disabled>
      </palette>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item rowspan="3" row="0" column="2" >
    <widget class="QScrollBar" name="fileScrollBar" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>0</hsizetype>
       <vsizetype>7</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip" >
      <string>Navigate in your file</string>
     </property>
     <property name="maximum" >
      <number>0</number>
     </property>
     <property name="tracking" >
      <bool>false</bool>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabSource" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabSourceSingle" >
      <property name="accessibleName" >
       <string/>
      </property>
      <attribute name="title" >
       <string>Singular</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="QTextEdit" name="txtSource" >
         <property name="tabChangesFocus" >
          <bool>true</bool>
         </property>
         <property name="undoRedoEnabled" >
          <bool>false</bool>
         </property>
         <property name="readOnly" >
          <bool>true</bool>
         </property>
         <property name="tabStopWidth" >
          <number>79</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QTabWidget" name="tabTarget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabTargetSingle" >
      <attribute name="title" >
       <string>Singular</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="QTextEdit" name="txtTarget" >
         <property name="whatsThis" >
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>txtSource</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
